Android X compatibility

Unable to build
android.useAndroidX=true
android.enableJetifier=true

version: 2.3.0

Issue:
```
/node_modules/react-native-snackbar/android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java:129: error: package android.support.design.R does not exist
22-Dec-2020 10:58:14	
22-Dec-2020 10:58:14	        TextView textView = (TextView) snackbarView.findViewById(android.support.design.R.id.snackbar_text);
```
SnackbarModule.java - 
Line 129
```TextView textView = (TextView) snackbarView.findViewById(android.support.design.R.id.snackbar_text);```

--- a/android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java
+++ b/android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java
@@ -126,15 +126,13 @@ public class SnackbarModule extends ReactContextBaseJavaModule {
         }
         View snackbarView = snackbar.getView();
 
-        TextView textView = (TextView) snackbarView.findViewById(android.support.design.R.id.snackbar_text);
-        textView.setMaxLines(numberOfLines);
-
         if (rtl && Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
             snackbarView.setLayoutDirection(View.LAYOUT_DIRECTION_RTL);
             snackbarView.setTextDirection(View.TEXT_DIRECTION_RTL);
         }
 
         TextView snackbarText = snackbarView.findViewById(com.google.android.material.R.id.snackbar_text);
+        snackbarText.setMaxLines(numberOfLines);
         snackbarText.setTextColor(textColor);
 
         if (font != null) {

